tasks/facial_landmark_detection/model.py

```diff
@@ -0,0 +1,169 @@
+                Conv2dNormActivation(inp, hidden_dim, kernel_size=1, norm_layer=norm_layer, activation_layer=nn.ReLU6)
+            )
+        layers.extend(
+            [
+                # dw
+                Conv2dNormActivation(
+                    hidden_dim,
+                    hidden_dim,
+                    kernel_size=kernel,
+                    stride=stride,
+                    groups=hidden_dim,
+                    norm_layer=norm_layer,
+                    activation_layer=nn.ReLU6,
+                ),
+                # pw-linear
+                nn.Conv2d(hidden_dim, oup, 1, 1, 0, bias=False),
+                norm_layer(oup),
+            ]
+        )
+        self.conv = nn.Sequential(*layers)
+        self.out_channels = oup
+        self._is_cn = stride > 1
+
+    def forward(self, x: Tensor) -> Tensor:
+        if self.use_res_connect:
+            return x + self.conv(x)
+        else:
+            return self.conv(x)
+
+# 
+# Adapted from torchvision MobileNetV2 to allow for different kernel sizes
+# to be passed in instead of default value of 3
+class CustomMobileNetV2(nn.Module):
+    def __init__(
+        self,
+        num_classes: int = 1000,
+        width_mult: float = 1.0,
+        inverted_residual_setting: Optional[List[List[int]]] = None,
+        round_nearest: int = 8,
+        block: Optional[Callable[..., nn.Module]] = None,
+        norm_layer: Optional[Callable[..., nn.Module]] = None,
+        dropout: float = 0.2,
+    ) -> None:
+        """
+        MobileNet V2 main class
+
+        Args:
+            num_classes (int): Number of classes
+            width_mult (float): Width multiplier - adjusts number of channels in each layer by this amount
+            inverted_residual_setting: Network structure
+            round_nearest (int): Round the number of channels in each layer to be a multiple of this number
+            Set to 1 to turn off rounding
+            block: Module specifying inverted residual building block for mobilenet
+            norm_layer: Module specifying the normalization layer to use
+            dropout (float): The droupout probability
+
+        """
+        super().__init__()
+        # _log_api_usage_once(self)
+
+        if block is None:
+            block = CustomInvertedResidual
+
+        if norm_layer is None:
+            norm_layer = nn.BatchNorm2d
+
+        input_channel = 32
+        last_channel = 1280
+
+        if inverted_residual_setting is None:
+            inverted_residual_setting = [
+                # t, c, n, s, k
+                [1, 16, 1, 1, 3],
+                [6, 24, 2, 2, 3],
+                [6, 32, 3, 2, 3],
+                [6, 64, 4, 2, 3],
+                [6, 96, 3, 1, 3],
+                [6, 160, 3, 2, 3],
+                [6, 320, 1, 1, 3],
+            ]
+
+        # only check the first element, assuming user knows t,c,n,s,k are required
+        if len(inverted_residual_setting) == 0 or len(inverted_residual_setting[0]) != 5:
```

Comment:
Perhaps `all([len(x) == 5 for x in inverted_residual_setting])`

Reply:
@yrajas needs to take a look.

Reply:
This was reused from the torchvision implementation. Makes sense to fix it. I'll do it in my PR.